python/ml_croissant/ml_croissant/_src/core/git_test.py

```diff
@@ -11,29 +11,35 @@
 from ml_croissant._src.core.git import is_git_lfs_file
 from ml_croissant._src.core.path import Path
 
-_GIT_LFS_CONTENT = """version https://git-lfs.github.com/spec/v1
+_GIT_LFS_CONTENT = lambda: """version https://git-lfs.github.com/spec/v1
 oid sha256:5e2785fcd9098567a49d6e62e328923d955b307b6dcd0492f6234e96e670772a
 size 309207547
 """
 
-_NON_GIT_LFS_CONTENT = """name,age
+_NON_GIT_LFS_CONTENT = lambda: """name,age
```

Comment:
Just asking, isn't it _GIT_NON_LFS ?

Reply:
git-lfs goes in one word (https://git-lfs.com). I wanted to convey this file is not a git-lfs file, because it doesn't have the proper header, so I called it non git-lfs.

Does it make sense? If not, we can always change later for a better wording.